### PR TITLE
Add update the ref.yaml to consume the right container image.

### PR DESCRIPTION
**Pull request by rpsene**
Signed-off-by: Rafael Peria de Sene <rpsene@br.ibm.com>

### CODE REVIEW

**Review by stevekuznetsov — APPROVED** [review]

**Comment by vrutkovs** [comment]
/lgtm cancel

`name of reference in file ci-operator/step-registry/upi/deprovision/powervs/upi-deprovision-powervs-ref.yaml should be upi-deprovision-powervs`

File: ci-operator/step-registry/upi/deprovision/powervs/upi-deprovision-powervs-ref.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ref:
-  as: upi-deprovision-powervs
-  from: upi-installer
+  as: upi-install-powervs
```
Comment (vrutkovs) [inline ci-operator/step-registry/upi/deprovision/powervs/upi-deprovision-powervs-ref.yaml:2]:
```suggestion
  as: upi-deprovision-powervs
```

**Comment by stevekuznetsov** [comment]
Subsumed into  #18197
/close

**Pull request closed without merging**